triage/2022-11-22.md

```diff
@@ -0,0 +1,225 @@
+# 2022-11-22 Triage Log
+
+A fairly quiet week with regressions unfortunately slightly outweighing regressions. There was not any particular change of much note. Many of the regressions were justifiable since they were for critical bug fixes.
```

Comment:
```suggestion
A fairly quiet week with regressions unfortunately slightly outweighing improvements. There was not any particular change of much note. Many of the regressions were justifiable since they were for critical bug fixes.
```